src/DacpacTool/DacpacTool.csproj

```diff
@@ -12,23 +12,20 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.SqlServer.DACFX" Version="150.4826.1" />
-    <PackageReference Include="System.CommandLine" Version="2.0.0-beta1.20427.1" />
-    <PackageReference Include="System.Security.Permissions" Version="4.7.0" />
+    <PackageReference Include="Microsoft.SqlServer.DACFX" Version="150.4897.1" />
+    <PackageReference Include="System.CommandLine" Version="2.0.0-beta1.20574.7" />
+    <PackageReference Include="System.Security.Permissions" Version="5.0.0" />
```

Comment:
Is this update required?

Reply:
Not necessarily no, but I'd rather stay up to date with our dependencies. Do you have a concern about these updates that I'm not aware of?

Reply:
I think it can be removed, actually

Reply:
You mean the System.Security.Permissions package?

Reply:
Yes, the DacFX package now brings in the correct, newer version

Reply:
Yup, seems to work just fine.